src/todt.c

```diff
@@ -388,6 +388,9 @@ dt_t **Expression_toDt(Expression *e, dt_t **pdt)
             {
                 case Tarray:
                     pdt = dtsize_t(pdt, n);
+                    pdt = dtabytes(pdt, 0, n * e->sz, p, (unsigned)e->sz);
```

Comment:
When I'd like to insert a `printf` to `case Tpointer`, the case fallthrough was problematic.
In other words, the case fallthrough tangles the two cases.
I think the two lines redundancy is better so the two cases will be independent.